src/builder-manifest.c

```diff
@@ -3802,12 +3786,57 @@ builder_manifest_install_dep (BuilderManifest *self,
 
   if (!builder_maybe_host_spawnv (NULL, NULL, 0, error, (const char * const *)args->pdata))
     {
-      g_autofree char *commandline = flatpak_quote_argv ((const char **)args->pdata);
-      g_prefix_error (error, "running `%s`: ", commandline);
```

Comment:
Why remove the commandline from the error message? Seems useful for debugging.

Reply:
I left it out before I could figure out what to do with multiple errors. Your suggestion below solves that.

Reply:
This is now added back.